kinode/src/state.rs

```diff
@@ -404,7 +404,7 @@ async fn bootstrap(
         let our_drive_name = [package_name, package_publisher].join(":");
         let pkg_path = format!("{}/vfs/{}/pkg", &home_directory_path, &our_drive_name);
         // delete anything currently residing in the pkg folder
-        fs::remove_dir_all(&pkg_path).await?;
+        let _ = fs::remove_dir_all(&pkg_path).await;
```

Comment:
I'd recommend doing the following instead. Swallowing `Result`s can lead to stuff that is hard to debug
```
let pkg_path = std::path::PathBuf::from(pkg_path);
if pkg_path.exists() {
    fs::remove_dir_all(&pkg_path).await?;
}
```